turbopack/crates/turbopack-ecmascript/src/references/pattern_mapping.rs

```diff
@@ -107,27 +107,15 @@ impl SinglePatternMapping {
         match self {
             Self::Invalid => self.create_id(key_expr),
             Self::Unresolvable(request) => throw_module_not_found_expr(request),
-            Self::Ignored => {
-                quote!("{}" as Expr)
-            }
-            Self::Module(_) | Self::ModuleLoader(_) => Expr::Call(CallExpr {
-                callee: Callee::Expr(quote_expr!("__turbopack_require__")),
-                args: vec![ExprOrSpread {
-                    spread: None,
-                    expr: Box::new(self.create_id(key_expr)),
-                }],
-                span: DUMMY_SP,
-                ..Default::default()
-            }),
-            Self::External(request, ExternalType::CommonJs) => Expr::Call(CallExpr {
-                callee: Callee::Expr(quote_expr!("__turbopack_external_require__")),
-                args: vec![ExprOrSpread {
-                    spread: None,
-                    expr: request.as_str().into(),
-                }],
-                span: DUMMY_SP,
-                ..Default::default()
-            }),
+            Self::Ignored => *quote_expr!("{}"),
```

Comment:
```suggestion
            Self::Ignored => quote!("{}" as Expr),
```

No need to create a box and destroy it again